subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/DependencyManagementBuildScopeServices.java

```diff
@@ -334,8 +337,10 @@ RepositoryBlacklister createRepositoryBlacklister() {
         return new ConnectionFailureRepositoryBlacklister();
     }
 
-    StartParameterResolutionOverride createStartParameterResolutionOverride(StartParameter startParameter) {
-        return new StartParameterResolutionOverride(startParameter);
+    StartParameterResolutionOverride createStartParameterResolutionOverride(StartParameter startParameter, BuildLayoutFactory buildLayoutFactory) {
```

Comment:
Can you use `BuildLayout` here?  It should be a build scope service created in this same way.

Reply:
Nope, `BuildLayout` is not available at this stage.